discovery/gossiper.go

```diff
@@ -2430,7 +2433,8 @@ func (d *AuthenticatedGossiper) handleNodeAnnouncement(nMsg *networkMsg,
 	// TODO(roasbeef): get rid of the above
 
 	log.Debugf("Processed NodeAnnouncement: peer=%v, timestamp=%v, "+
```

Comment:
I'll try get around to updating the gossiper to use structured logging. That way we only need to add all this to the context once and then can just `log.DebugS(ctx, "Processed NodeAnnouncement")`